content/chapters/compute/lab/support/wait-for-me/wait_for_me_threads.c

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
```

Comment:
```suggestion
// SPDX-License-Identifier: BSD-3-Clause
```
This is the convention for `.c` files. `.h` files should use `/* ... */`: https://www.kernel.org/doc/html/v5.0/process/license-rules.html#license-identifier-syntax